include/eve/detail/function/simd/x86/bitmask.hpp

```diff
@@ -11,74 +11,109 @@
 #pragma once
 
 #include <eve/detail/abi.hpp>
-#include <eve/detail/function/bit_cast.hpp>
-#include <eve/detail/meta.hpp>
 #include <bitset>
 
 namespace eve::detail
 {
   //================================================================================================
-  // Logical to Bitmap - use movemask variant
+  // Logical to Bits
   //================================================================================================
   template<typename T, typename N, x86_abi ABI>
-  EVE_FORCEINLINE auto to_bitmap( sse2_ const&, logical<wide<T, N, ABI>> const& p ) noexcept
+  EVE_FORCEINLINE auto to_bits( sse2_ const&, logical<wide<T, N, ABI>> const& p ) noexcept
   {
-    using type = std::bitset<N::value>;
+    using type = typename logical<wide<T, N, ABI>>::bits_type;
+    return bit_cast(p.mask(), as_<type>{});
+  }
```

Comment:
Do you need a `bit_cast`? Instead of just accessing the member?

Reply:
`logical<wide<T>>` mask() returns a `wide<T>`. bits() returns the same wide but as a wide of integral values. Hence the bitcast. The one retuning the mask directly is to_bitmap.